### PR TITLE
Update language to include other underrepresented groups

### PR DESCRIPTION
**Pull request by mxie**
We've expanded our audience to more than just women!

Addresses #44.

### CODE REVIEW

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -11,16 +11,16 @@
...
+    <p>Led by an all-volunteer team of seasoned, enthusiastic Ruby and Rails developers, the workshop introduces people of all backgrounds to the concepts, tools, and techniques of Ruby and Rails development. Our audience is those with no or little programming experience.</p>
     <p>We welcome you to the Boston Ruby community. Whatever your goal is in learning to program, we hope to connect you with the tools to take the next step.</p>
   </section>
 
   <section class="promoting-diversity">
     <h2 class="section-title">Promoting Diversity</h2>
-    <p>The Boston software scene needs more women. So this workshop is for women and their guests. You should identify as a woman, or be the male guest of a woman who is attending.</p>
+    <p>The Boston software scene needs more people with diverse backgrounds, so this workshop is for those who are in underrepresented groups in tech. These include, but are not limited to, those who identify as women, people of color, LGBTQ, military veterans, and more. If you do not fall under these or similar groups, you can still join us as someone's guest.</p>
```
Comment (edwardloveall) [inline app/views/pages/about.html.erb]: This whole line feels a bit clunky to me. Here's another stab at it:

> The Boston software scene needs more people from diverse backgrounds. RailsBridge is for those underrepresented groups. This includes, but is not limited to, those who identify as women, people of color, LGBTQ, military veterans, and more.

Comment (galiat) [inline app/views/pages/about.html.erb]: I like that. 

Are you suggesting we omit the part about guests @edwardloveall? I would prefer to keep it, maybe reworded. 

I like that you & @mxie managed to phase that without repeating 'underrepresented in tech' several times.

Comment (edwardloveall) [inline app/views/pages/about.html.erb]: @galiat Agreed. I skipped it while rewriting.

>The Boston software community needs more people from diverse backgrounds. RailsBridge is for those underrepresented groups. This includes, but is not limited to, those who identify as women, people of color, LGBTQ, military veterans, and more. If you do not fall under these or similar groups, you are still welcome to come as someone's guest.

Comment (mxie) [inline app/views/pages/about.html.erb]: I like that a lot better. Thanks @edwardloveall! Updated in b325b84.

**Comment by mxie** [comment]
Addressed the feedback and added another change to changes "mothers" to "parents" so that it's more inclusive.

File: app/assets/stylesheets/_about.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@
   }
 }
 
-.empowering-women,
```
Comment (galiat) [inline app/assets/stylesheets/_about.scss]: very thorough

**Comment by edwardloveall** [comment]
Looks good to me!

File: app/assets/stylesheets/_about.scss (hunk 2 above)
Comment (mxie) [inline app/assets/stylesheets/_about.scss]: Why thank you :D
